lib/active_job/jobs_relation.rb

```diff
@@ -274,6 +276,7 @@ def satisfy_filter?(job)
     end
 
     def filters
+      # @filetrs ||= queue_adapter.supported_job_filters(self).select { |property| public_send(property).present? && !queue_adapter.natively_supported_job_filters?(self, property) }
```

Comment:
```suggestion
      # @filetrs ||= queue_adapter.supported_job_filters(self).select { |property| public_send(property).present? && !queue_adapter.supports_job_filter_natively?(self, property) }
```
To be consistent with the other method, `supports_job_filter?`.